graphql/chat/mutations.ts

```diff
@@ -53,8 +53,8 @@ export class MutateChatResolver {
         };
 
         if (allowed) {
-            await getOrCreateConversation([user, participantUser], conversationInfos);
-            return true;
+            const conversation = await getOrCreateConversation([user, participantUser], conversationInfos);
+            return conversation.id;
         }
         throw new Error('Participant is not allowed to create conversation.');
```

Comment:
Isn't this the same change as in this PR https://github.com/corona-school/backend/pull/655?